google/cloud/dataproc_spark_connect/session.py

```diff
@@ -46,9 +47,10 @@
     TerminateSessionRequest,
 )
 from google.cloud.dataproc_v1.types import sessions
+from pyspark.sql.connect.client.core import SparkConnectClient
 from pyspark.sql.connect.session import SparkSession
 from pyspark.sql.utils import to_str
-from typing import Any, cast, ClassVar, Dict, Optional
+from typing import Any, cast, ClassVar, Dict, Optional, Union
```

Comment:
I didn't catch this earlier, but this import is out of place. It should be moved up to line 25 since it is part of the standard library and so should be included in the same import grouping as the other standard library imports.